### PR TITLE
feature: re-arrange my account dialog

### PR DESCRIPTION
**Pull request by lizable**
* related: #2083 

- required Core version: `24.03.0` or newer version
   - You need to upgrade alembic version to the latest by executing the following command in Core directory: 
      ```
       ./py -m alembic upgrade head
      ```

In this PR, keypair in user account information has been moved to user settings menu, since keypair is not maintained by user, but only for admin, which may confuses user if the fields is remaining in user account modal. 
Therefore, there are two things to check during this PR review:
- [x] Check access key and secret key is removed from user information(so called "My Account" modal)
- [x] Check the new menu "My keypair Info" in user settings menu
- [x] Check all the keypairs of current user in "My keypair info modal"
- [x] Check the main access key tag is attached correctly.

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [x] Minimum required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [x] Test case(s) to demonstrate the difference of before/after

### Screenshot(s)
Screenshot 2023-12-07 at 6 41 56 PM
Screenshot 2023-12-07 at 6 41 41 PM
Screenshot 2023-12-07 at 7 36 14 PM

### CODE REVIEW

File: react/src/components/KeypairInfoModal.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        columns={[
+          {
+            title: '#',
+            fixed: 'left',
+            render: (id, record, index) => {
+              ++index;
+              return index;
+            },
+            showSorterTooltip: false,
+          },
```
Comment (agatha197) [inline react/src/components/KeypairInfoModal.tsx]:
```suggestion
            showSorterTooltip: false,
            rowScope: 'row',
          },
```

Comment (agatha197) [reply react/src/components/KeypairInfoModal.tsx]: control panel use `rowScope` to the index column. I added it to the style guide (KB).
ref: https://ant.design/components/table#components-table-demo-colspan-rowspan

File: react/src/components/KeypairInfoModal.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            dataIndex: 'access_key',
+            fixed: 'left',
+            render: (value) => (
+              <Flex direction="column" align="start">
+                <Typography.Text ellipsis copyable style={{ width: 150 }}>
+                  {value}
+                </Typography.Text>
+                {supportMainAccessKey && value === user?.main_access_key ? (
+                  <Tag color="red">{t('credential.MainAccessKey')}</Tag>
+                ) : null}
```
Comment (agatha197) [inline react/src/components/KeypairInfoModal.tsx]:
```suggestion
                {supportMainAccessKey && value === user?.main_access_key && (
                  <Tag color="red">{t('credential.MainAccessKey')}</Tag>
                )}
```

**Comment by lizable** [comment]
Just updated few things:
- Refresh(fetching) keypair every time the modal opens.
- Enlarge Keypair info modal to show both access key and secret key clearly, with out ellipsis.

## Screenshot(s)
Screenshot 2023-12-08 at 12 38 48 PM

**Pull request merged**